FourFunctionCalculator added

VaibhavBansal93 opened the pull request:
Have added the FourFunctionCalculator program written in python language.
Please accept my pull request.
I'll be glad to hear back regarding any feedbacks and improvement in my code.

grufelous commented:
Hi @VaibhavBansal93! Great work with the succinct programs. Please submit this as two separate PRs, and in case you don't find a question for the same, then create a PR for that too. As for the `FourFunctionCalculator`, change it to use Python3 and not Python2, which is being actively phased out.

VaibhavBansal93 commented:
Hello @grufelous! Thanks a lot for your feedback.
I have updated the FourFunctioncalculator to Python3.
I have also removed the ReverseInteger file for now ( will make a new PR for it)
